pkgs/servers/mpd/i386.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/src/decoder/plugins/FfmpegDecoderPlugin.cxx b/src/decoder/plugins/FfmpegDecoderPlugin.cxx
+index 6986453..167fc07 100644
+--- a/src/decoder/plugins/FfmpegDecoderPlugin.cxx
++++ b/src/decoder/plugins/FfmpegDecoderPlugin.cxx
+@@ -20,8 +20,8 @@
+ /* necessary because libavutil/common.h uses UINT64_C */
+ #define __STDC_CONSTANT_MACROS
+ 
+-#include "lib/ffmpeg/Time.hxx"
+ #include "config.h"
++#include "lib/ffmpeg/Time.hxx"
```

Comment:
No, their imports are order-dependent. That is, on `i686` systems the software will not work correctly if `config.h` is not imported first. That is checked by a file `check.h`, and only if it’s a non-release build and it’s compiled for `i686`.

I point you to the thread at http://mailman.blarg.de/pipermail/mpd-devel/2017-February/000601.html for further incompetence.